src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -257,7 +257,7 @@ object DepositDir {
     for {
       _ <- Try { depositDir.createDirectories }
       bag <- DansV0Bag.empty(depositDir / "bag")
-      _ = bag.withEasyUserAccount(deposit.user)
+      _ = bag.withCreated()
```

Comment:
You deleted `withEasyUserAccount` here. May I assume this `deposit.user` is still listed in the `deposit.properties` as `depositor.userId`? That's where `easy-ingest-flow` gets its data from to add the user account...